src/main/java/controller/AllProductServlet.java

```diff
@@ -0,0 +1,22 @@
+package controller;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+@WebServlet(value = "/allProducts")
+public class AllProductServlet extends HttpServlet {
+
+  @Override
+  protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
```

Comment:
https://mate-academy.github.io/style-guides/java/java.html#s4.4-column-limit
in all classes